routes/api/users.js

```diff
@@ -57,4 +66,29 @@ router.get("/current", auth, async (req, res, next) => {
 	res.send({ email, subscription });
 });
 
+router.patch(
+	"/avatars",
+	auth,
+	upload.single("avatar"),
+	async (req, res, next) => {
+		const userId = req.user._id;
+		try {
+			const originalFileName = req.file.originalname;
```

Comment:
Te indentacje są zbyt duże, powinny być 2 spaces or 1 tab
```suggestion
		try {
		  const originalFileName = req.file.originalname;
```